arm-redis/2016-04-01/swagger/redis.json

```diff
@@ -794,6 +913,98 @@
         "files"
       ],
       "description": "Parameters for redis import operation."
+    },
+    "ScheduleEntry": {
+      "properties": {
+        "dayOfWeek": {
+          "type": "string",
+          "description": "Day of week when cache can be patched.",
+          "enum": [
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday",
+            "Sunday"
+          ],
+          "x-ms-enum": {
+            "name": "DayOfWeek",
+            "modelAsString": true
+          }
+        },
+        "startHourUtc": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Start hour after which cache patching can start."
+        },
+        "maintenanceWindow": {
+          "type": "string",
+          "format": "duration",
+          "description": "ISO8601 timespan specifying how much time cache patching can take. "
+        }
+      },
+      "required": [
+        "dayOfWeek",
+        "startHourUtc"
+      ]
+    },
+    "ScheduleEntries": {
+      "properties": {
+        "scheduleEntries": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ScheduleEntry"
+          },
+          "description": "List of patch schedules for redis cache."
+        }
+      },
+      "required": [
+        "scheduleEntries"
+      ],
+      "description": "List of patch schedules for redis cache."
+    },
+    "RedisPatchSchedulesRequest": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ScheduleEntries",
```

Comment:
You are referring this as ScheduleEntries. I did not find any definition with that name. In this commit the newly added definition is ScheduleEntry. Please make sure that the naming is correct.
@tbombach - We should make sure we add this check as well in the validation rules.